doc/DoxygenLayout.xml

```diff
@@ -0,0 +1,196 @@
+<doxygenlayout version="1.0">
+  <!-- Generated by doxygen 1.8.11 -->
+  <!-- Navigation index tabs for HTML output -->
+  <navindex>
+    <tab type="user" visible="yes" title="EDM4hep Home Page" url="https://cern.ch/edm4hep"/>
```

Comment:
This website doesn't exist, maybe pointing to edm4hep.gihub.io would be better?

Reply:
Hmmm, I did create `cern.ch/EDM4hep`, and redirected it to the gh-page for now,  but something seems to have gone wrong with the capitalization. I'll look into it.

Reply:
Right, I get 403 (forbidden), not 404 (not found)

Reply:
Fixed. (Redirected to my fork for now, since this repository is not deployed)